deps.bzl

```diff
@@ -31,10 +31,10 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 rules_scala_version = "6c16cff213b76a4126bdc850956046da5db1daaa"
 
-rules_haskell_version = "76f7b1c7328f59bf2f38b56bce0036153c65a40d"
-rules_haskell_sha256 = "5553f76d40db672b499a76fd56fab75df6fe465531f2eedd7da39a3deaed54a6"
-rules_nixpkgs_version = "c966bb8bd335f1e244c03efe6e7a1afa9784038e"
-rules_nixpkgs_sha256 = "ccafea4fc4d5fa2ddba2882f76728558bfe2c12657f7f56078ece43a31761148"
+rules_haskell_version = "ac87721a4dbc0f7dbe731df928d322f02ed93330"
```

Comment:
Do we need the bump here or is this unrelated? Happy to include it either way, I’d just like to understand it.

Reply:
Yes, the `rules_nixpkgs` update includes a change to the nixpkgs platform constraints. That requires an update to `rules_haskell` as well. See https://github.com/tweag/rules_nixpkgs/pull/122 and https://github.com/tweag/rules_haskell/pull/1313/files#diff-3180f021208ca49e05f50da55d00dd65R15